src/components/ebay-wizard-stepper/README.md

```diff
@@ -0,0 +1,32 @@
+# ebay-wizard-stepper
+
+## ebay-wizard-stepper Example
+
+```marko
+<ebay-wizard-stepper>
+    <@step>Started</@step>
+    <@step>Shipped</@step>
+    <@step current>Transit</@step>
+    <@step>Delivered</@step>
+</ebay-wizard-stepper>
+```
+
+## ebay-wizard-stepper Sub-tags
+
+Tag | Required | Description
+--- | --- | ---
+`<@step>` | Yes | Each step to represented. The body will display as text next to the current state
+
+## ebay-wizard-stepper Attributes
+
+Name | Type | Stateful | Required | Description
+--- | --- | --- | --- | ---
+`vertical` | Boolean | No | No | If true, then renders vertical version of stepper
```

Comment:
`current` makes sense as a boolean, as it's an "off" or "on" state, but I personally would prefer an enum for layout. I'd like us to be more in line with flexbox and grid layout. So instead of `vertical`, I suggest `direction` with values of `column` (default) or `rows`. That keeps us forwards compatible for things like `column-reverse` or `row-reverse` should we ever need it.

Reply:
Should default be `column`? I assume the column is the top down one, is that considered the default?